core/cmd/app.go

```diff
@@ -354,6 +354,18 @@ func NewApp(client *Client) *cli.App {
 						},
 					},
 				},
+				{
+					Name:   "validate",
+					Usage:  "Validate provided TOML config file",
+					Action: client.ConfigFileValidate,
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:     "file, f",
+							Usage:    "TOML config `FILE` name to validate",
+							Required: true,
+						},
+					},
```

Comment:
Can the global `--config` flag be used instead of introducing another one? 
Then the `client.Config` is already available from the shared 'before' routine.